### ISSUE TITLE
Can you add this please?

### ISSUE THREAD

**Issue by fastsol** [adbario/slim-secure-session-middleware#11]
It would be helpful to have a method to get a list of messages from a namespace.  Currently all we can do is get a single item by it's name, but not all under the namespace, unless I'm missing something in my understanding of how it works.

**Comment by adbario** [comment]
You're absolutely right with this and that option is coming for next major version. I have no estimate for schedule yet though.

**Comment by fastsol** [comment]
Just checking if you have an update on this.  I'm revisiting some projects and it would be helpful to have this feature added.  Thanks!

**Comment by adbario** [comment]
Really sorry about this, just haven't had time to work on this yet. If you need a fast fix, you could store your messages one level deeper inside namespace and then you could select them all by the parent key.
